core/lib/Layout.re

```diff
@@ -257,47 +257,47 @@ module Create = (Node: Flex.Spec.Node, Encoding: Flex.Spec.Encoding) => {
       {
         ...style,
         positionType,
-        left: !isUndefined(left) ? int_of_scalar(left) : style.left,
-        top: !isUndefined(top) ? int_of_scalar(top) : style.top,
-        right: !isUndefined(right) ? int_of_scalar(right) : style.right,
-        bottom: !isUndefined(bottom) ? int_of_scalar(bottom) : style.bottom,
+        left: !isUndefined(left) ? left : style.left,
+        top: !isUndefined(top) ? top : style.top,
+        right: !isUndefined(right) ? right : style.right,
+        bottom: !isUndefined(bottom) ? bottom : style.bottom,
       };
-    | `Flex(f) => {...style, flex: int_of_scalar(f)}
+    | `Flex(f) => {...style, flex: f}
     | `FlexDirection(flexDirection) => {...style, flexDirection}
-    | `FlexGrow(f) => {...style, flexGrow: int_of_scalar(f)}
-    | `FlexShrink(f) => {...style, flexShrink: int_of_scalar(f)}
-    | `FlexBasis(f) => {...style, flexBasis: int_of_scalar(f)}
+    | `FlexGrow(f) => {...style, flexGrow: f}
+    | `FlexShrink(f) => {...style, flexShrink: f}
+    | `FlexBasis(f) => {...style, flexBasis: f}
```

Comment:
`` `FlexGrow(flexGrow) => {...style, flexGrow}``? And others, too.

I used to use short because I had also to convert the value (bikeshed for better readability).

Reply:
ok